lib/src/controls/better_player_material_controls.dart

```diff
@@ -503,6 +504,13 @@ class _BetterPlayerMaterialControlsState
     );
   }
 
+  Widget _buildAirplayButton() => AnimatedOpacity(
```

Comment:
pamietaj ze ten feature tylko. na iOS ma byc, a teraz przycisk sie wyswietli na obu platformach?

Reply:
swoją drogą jak to działą? Nie widze implementacji innej oprocz przycisku :P